ui/app/utils/default_jobs/hello-world.js

```diff
@@ -0,0 +1,38 @@
+export default `job "hello-world" {
+  # Specifies the datacenters within which this job should be run.
+  # Leave as "dc1" for the default datacenter.
+  datacenters = ["dc1"]
```

Comment:
Ugh, I just realized the datacenters will probably mess up every job

Maybe we can get datacenter globbing into 1.5 for this. I was on the fence on trying to get this one in, but it'd make this feature much nicer.